src/main/java/reposense/util/FileUtil.java

```diff
@@ -38,6 +39,56 @@
     private static final String GITHUB_API_DATE_FORMAT = "yyyy-MM-dd";
     private static final ByteBuffer buffer = ByteBuffer.allocate(1 << 11); // 2KB
 
+    /**
+     * Zips all the relative folders and relevant files of {@code fileTypes} contained in
+     * {@code sourceAndOutputPath} directory into the same folder.
+     * @param relativePaths contains the relevant folders to be zipped.
+     * @param sourceAndOutputPath contains the directory where the relevant folders and files are located at and
+     *                           where to be zipped to.
+     * @param fileTypes contains the file types to be zipped. Only files which are of the type "fileTypes" will be
+     *                  zipped.
+     */
+    public static void zipRelativeFiles(HashSet<Path> relativePaths,
+            Path sourceAndOutputPath, String... fileTypes) {
+        zipRelativeFiles(relativePaths, sourceAndOutputPath, sourceAndOutputPath, fileTypes);
+    }
+
+    /**
+     * Zips all the relevant files and relative folders
+     * @param relativePaths contains the relevant folders to be zipped.
+     * @param sourcePath contains the directory where the relevant folders and files are located at
+     * @param outputPath contains the directory to be zipped to.
+     * @param fileTypes contains the file types to be zipped. Only files which are of the type "fileTypes" will be
+     *                  zipped.
+     */
+    public static void zipRelativeFiles(HashSet<Path> relativePaths,
```

Comment:
FileUtil#zip is similar to FileUtil#zipRelativeFiles and its not used anymore right?

Reply:
Yup, FuleUtil#zip method is no longer used anymore

Reply:
Then, we should remove this method from code base.

Reply:
Ok, will make the changes